baseimages/components/ansible.yml.tftpl

```diff
@@ -0,0 +1,84 @@
+---
+name: ${ami}_ansible
+description: Install ansible roles for ${ami}
+schemaVersion: 1.0
+parameters:
+  - Version:
+      type: string
+      default: ${version}
+      description: Uses the recipe version to avoid any issues with the branch template arg.
+  - Platform:
+      type: string
+      default: "Linux"
+      description: Platform.
+phases:
+  - name: build
+    steps:
+      # Install ansible roles
+      - name: InstallAndRunAnsible
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -e
+              repo="modernisation-platform-ami-builds"
+              ami_tag="${ami}"
+
+              # clone ansible roles and playbook
+              ansible_dir=$(mktemp -d)
+              echo "Cloning $repo into $ansible_dir using branch=${branch}"
+              cd $ansible_dir
+              git clone -b ${branch} "https://github.com/ministryofjustice/$repo.git"
+
+              # set python version
```

Comment:
We have agreed in the past on the specific python version (major.minor.patch) so not sure why this has been dropped; different version of python can cause other steps to fail (e.g. installation of a specific version of pip or ansible); also if the python version does not exist (the OS' python version is different from what you want to use), then another python version should be installed on the machine; it is not a good practice to rely on the OS' python version anyway

Reply:
scrap that, I think this component will be used in multiple AMIs (RHEL7, RHEL6), hence the different Python version, it makes sense to me now :)